spyder/plugins/application/widgets/status.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+Status widget for Spyder updates.
+"""
+
+# Standard library imports
+import logging
+
+# Third party imports
+from qtpy.QtCore import Slot
+
+# Local imports
+from spyder import get_versions
+from spyder.api.widgets.status import StatusBarWidget
+from spyder.config.base import _, is_pynsist
+from spyder.plugins.application.widgets.install import (
+    UpdateInstallerDialog, NO_STATUS, DOWNLOADING_INSTALLER, INSTALLING,
+    FINISHED, PENDING, CHECKING, CANCELLED)
+from spyder.utils.icon_manager import ima
+
+logger = logging.getLogger(__name__)
+
+
+class ApplicationUpdateStatus(StatusBarWidget):
+    """Status bar widget for Application update status."""
+    BASE_TOOLTIP = _("Application update status")
+    ID = 'application_update_status'
+
+    def __init__(self, parent):
+
+        self.tooltip = self.BASE_TOOLTIP
+        super().__init__(parent, show_spinner=True)
+
+        # Installation dialog
+        self.installer = UpdateInstallerDialog(self)
+
+        self.sig_clicked.connect(self.show_installation_dialog)
```

Comment:
Lets change the method name from `show_installation_dialog` to `show_installation_dialog_or_menu`:
```suggestion
        self.sig_clicked.connect(self.show_installation_dialog_or_menu)
```